### PR TITLE
Improve logging and adapt to the new hyper logger exposure.

### PR DESCRIPTION
**Pull request by Pchelolo**
1. Stop using hyper.log and use the logger instead.
2. Include the rule name in all the logs
3. Include the topic name in the log whenever possible
4. Fix up some logging bugs.
5. Stop including the rule name in the log level, instead, make the log level descriptive on what's happened and include the rule name as a separate property.

Depends on the hyperswitch 0.10 changes
cc @wikimedia/services

### CODE REVIEW

**Comment by d00rman** [comment]
Needs a rebase now. Figures :)

**Comment by Pchelolo** [comment]
Done with a rebase

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "bluebird": "^3.5.1",
     "cassandra-uuid": "^0.0.2",
     "extend": "^3.0.1",
-    "hyperswitch": "^0.9.6",
+    "hyperswitch": "^0.10.0",
```
Comment (d00rman) [inline package.json]: Since 0.10.1 is already out, let's require it.

Comment (Pchelolo) [reply package.json]: Done

**Pull request merged**